.expeditor/verify_private.pipeline.yml

```diff
@@ -137,7 +138,6 @@ steps:
           environment:
             - SEMGREP_TOKEN
             - SEMGREP_ID
-    soft_fail: true
```

Comment:
Meant to remove this earlier; since we are now running differential scans--scanning only files in the current PR--it is safe to have hard failures for any semgrep findings.